docs/user_guide/security/identity_security.rst

```diff
@@ -0,0 +1,503 @@
+
+See :ref:`command_categories` for command categories.
+
+Controls and Conditions
+^^^^^^^^^^^^^^^^^^^^^^^
+A *control* is a set of one or more conditions that is specified in the permission matrix. Conditions specify relationships among the subject user, the site, and the job submitter. The following are supported relationships:
+
+    - The user belongs to the site's organization (user org = site org)
+    - The user is the job submitter (user name = submitter name)
+    - The user and the job submitter are in the same org (user org = submitter org)
+    - The user is a specified person (user name = specified name)
+    - The user is in a specified org (user org = specified org)
+
+Keep in mind that the relationship is always relative to the subject user - we check to see whether the user's name or org has the right relationship with the site or job submitter.
+
+Since conditions need to be expressed in the policy definition file (authorization.json), some concise and consistent notations are needed. The following are the notations for these conditions:
+
+.. csv-table::
+    :header: Notation,Condition,Examples
+    :widths: 15, 20, 15
+
+    o:site,The user belongs to the site's organization
+    n:submitter,The user is the job submitter
+    o:submitter,The user and the job submitter belong to the same org
+    n:<person_name>,The user is a specified person,n:john@nvidia.com
+    o:<org_name>,The user is in a specified org,o:nvidia
+
+The words "site" and "submitter" are reserved.
+
+In addition, two words are used for extreme conditions:
+
+    - Any user is allowed: any
+    - No user is allowed: none
+
+See :ref:`sample_auth_policy` for an example policy.
+
+Policy Evaluation
+^^^^^^^^^^^^^^^^^
+Policy evaluation is to answer the question: is the user allowed to do this command? 
+
+The following is the evaluation algorithm:
+
+    - If a control is defined for this command and user role, then this control will be evaluated;
+    - Otherwise, if the command belongs to a category and a control is defined for the category and user role, then this control will be evaluated;
+    - Otherwise, return False
+
+As a shorthand, if the control is the same for all rights for a role, you can specify a control for a role without explicitly specifying rights one by one. For example, this is used for the "project_admin" role since this role can do everything.
+
+Command Authorization Process
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+We know that users operate NVFLARE systems with admin commands via the FLARE Console. But when a user issues a command, how does authorization happen
+throughout the system? In NVFLARE 2.1 and before, the authorization policy was evaluated and enforced by the FL Server that processes the command.
+
+In NVFLARE 2.2 and after, the command is still received by the FL Server.
```

Comment:
maybe we don't need to mention 2.1 VS 2.2 here?
it might cause confusion.

Reply:
cleaned this up